modules/DMZ/data_kit/filler_strategy.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+import dataset_insight
+import data_filler
+
+class FillerStrategy(object):
+
+    def __init__(self):
+        self.file_name = None
+        self.avg_thresh = None
+        self.value_thresh = None
+        self.fill_value = None
+        self.pandas_dataset = None
+        self.missing_ratios_dict = None
+
+        self.get_config_options()
+        self.get_dataset()
+        self.get_missing_ratios_dict()
+        self.run_fillers()
+
+    def get_config_options(self):
+        self.file_name = "titanic_train.csv"
```

Comment:
Doesn't matter yet, we're waiting on the config tool. There could be a comment next to it with a TODO to fix though.